test/indicators/flash-message-container.test.js

```diff
@@ -1,70 +1,74 @@
 import React from 'react'
-import { mount } from 'enzyme'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { within } from '@testing-library/dom'
 import { FlashMessageContainer } from '../../src'
 
 const successMessage = {
   id: '0',
   message: 'Success!',
   isError: false,
-  props: {},
+  props: {
+    'data-testid': 'success'
+  },
 }
 const failureMessage = {
   id: '1',
   message: 'Failure!',
   isError: true,
-  props: {},
+  props: {
+    'data-testid': 'failure'
+  },
 }
 
-test('FlashMessageContainer displays all provided redux-flash messages', () => {
-  const wrapper = mount(
+test('FlashMessageContainer displays all provided flash messages', () => {
+  render(
     <FlashMessageContainer messages={[successMessage, failureMessage]} />
   )
-  expect(wrapper.find('div.flash-message.success').exists()).toBe(true)
```

Comment:
I didn't migrate this assertion because I think it's actually testing the `FlashMessage` component, which we have separate tests for! If the class name was changed, I don't think this test should fail (but the spec in `flash-message.test.js` should)

Reply:
That makes sense 👍